lava/magma/core/decorator.py

```diff
@@ -104,62 +104,63 @@ def decorate_process_model(cls: type):
     return decorate_process_model
 
 
-def tags(*args: ty.Union[str, ty.List[str]]):
+def tag(*args: ty.Union[str, ty.List[str]]):
```

Comment:
Actually, we should not use raw strings but 'official' enums like in the case of Resources. Ensure, that we don't just tag ProcModels with random strings but with tags that have RunCfg support.

Reply:
I will open a new issue for this and this will be a feature improvement over the current first implementation